packages/vue/src/components/atoms/SfBullets/SfBullets.js

```diff
@@ -6,14 +6,14 @@ export default {
      */
     total: {
       type: Number,
-      default: 0
+      default: 3
     },
     /**
-     * Index of the curretly active bullet
+     * Index of the currently active bullet (0-indexed)
      */
     current: {
       type: Number,
-      default: 0
+      default: 1
```

Comment:
```suggestion
      default: 0
```

Imo the natural is start from the first bullet, right?

Reply:
I mentioned it in the commit message: It doesn't work with `0`, because then the computed value will be negative (`total - 1 - current`).
IMHO we should set only _reasonable_ default values, i.e. don't set default values if the user will very likely change them anyway. In that case it's even better to make the props `required`.

Reply:
It'll be negative only if current is bigger than `total - 1`, not the case when `current` is zero, right? I think you mean `total` can't be zero, not?

`current` is the index, so I really think it should be the first by default, if it fires an error when `current` is zero I think it's a bug and we must fix, don't you agree?

About reasonable default values, I agree with you, but I think it's the case of `total` prop on this component, not `current`, wdyt?